server.py

```
@@ -448,6 +450,39 @@ def extract_passthrough_headers(request: Request) -> dict:
+                    if not cancel_event.is_set():
+                        cancel_event.set()
+                        logging.info(
+                            "stream %s: HTTP client disconnected, cancelling", label
+                        )
+                    return
+        except asyncio.CancelledError:
+            pass
+
+    try:
+        _anyio_from_thread.run_sync(lambda: asyncio.ensure_future(_watch()))
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)